### ISSUE TITLE
When trying to connect to ZeroMap, I get a Server Error

### ISSUE THREAD

**Issue by TheNain38** [HelloZeroNet/ZeroNet#219]
When connecting to [ZeroMap](http://127.0.0.1:43110/1yUji4qPD7GAog8XkP3EDMEfrL995S4Xd), I get a Server error:
````
Err: UnicodeDecodeError: 'ascii' codec can't decode byte 0xe9 in position 84: ordinal not in range(128) in UiServer.py line 75 > UiRequest.py line 81 > UiRequest.py line 200 > SiteManagerPlugin.py line 52 > SiteManager.py line 57 > Site.py line 50 > SiteStorage.py line 26 > SiteStorage.py line 221 > SiteStorage.py line 240
````

**Comment by HelloZeroNet** [comment]
The site does not have any special characters in the files, so probably you have installed your zeronet to non-english characters directory

try moving your zeronet installation to \ZeroNet and try again

**Comment by TheNain38** [comment]
It works correctly, thanks. None the less it is a bug if it can't work correctly when having a folder with non-english character?

**Comment by HelloZeroNet** [comment]
Yes, its a bug, thanks for reporting, i will fix it soon

**Comment by HelloZeroNet** [comment]
Fixed in 3d558a4edfe527450d99458f10cf7681eb96590f
